test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -358,16 +358,20 @@ void test_ucp_tag_xfer::test_xfer_iov(size_t size, bool expected, bool sync,
 
     ucs::fill_random(sendbuf.begin(), sendbuf.end());
 
-    UCS_TEST_GET_BUFFER_DT_IOV(send_iov, send_iovcnt, sendbuf.data(), sendbuf.size(), iovcnt);
-    UCS_TEST_GET_BUFFER_DT_IOV(recv_iov, recv_iovcnt, recvbuf.data(), recvbuf.size(), iovcnt);
-
-    size_t recvd = do_xfer(&send_iov, &recv_iov, iovcnt, DATATYPE_IOV, DATATYPE_IOV,
-                           expected, sync, truncated);
+    ucp::data_type_desc_t send_dt_desc(DATATYPE_IOV, sendbuf.data(),
+                                       sendbuf.size(), iovcnt);
+    ucp::data_type_desc_t recv_dt_desc(DATATYPE_IOV, recvbuf.data(),
+                                       recvbuf.size(), iovcnt);
+
+    size_t recvd = do_xfer(send_dt_desc.buf(), recv_dt_desc.buf(), iovcnt,
```

Comment:
maybe use dt_sesc.count() instead of iovcnt?

Reply:
In this particular case,  `iovcnt` is common for `send_dt_desc` and `send_dt_desc` descriptors, this is why i pass it.
Where is more opportunities for refactoring, this PR adds minimal changes in existing tests.